### ISSUE TITLE
SAC with torch 1.5.1 shows some error

### ISSUE THREAD

**Issue by MrSyee** [MrSyee/pg-is-all-you-need#30]
Citrusboy1997 send me email:

> When I run the SAC code of your pg-is-all-you-need repository with Torch 1.5.1 or nightly version, an error shows that there is an inplace operation during the gradient computation.
According to this disscusion(https://discuss.pytorch.org/t/solved-pytorch1-5-runtimeerror-one-of-the-variables-needed-for-gradient-computation-has-been-modified-by-an-inplace-operation/90256?u=citrusboy1997), I solved this problem by putting all the optimizer.step() functions at the end of update_model() of the SACAgent class. I hope this can help you improve your code.

**Comment by Curt-Park** [comment]
Is this fixed?

**Comment by MrSyee** [comment]
Not yet. But I already know how to fix it. I do that soon.
